cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceContextConfig.java

```diff
@@ -126,6 +130,15 @@ public String getWorkflowBatch() {
     return jobContext.getConfiguration().get(HCONF_ATTR_WORKFLOW_BATCH);
   }
 
+  private void setAdapterName(@Nullable String adapterName) {
```

Comment:
probably not a good idea to let setter to pass null. Caller to make sure not to pass null when trying to set adapter name to context config.